.doc_gen/metadata/medical-imaging_metadata.yaml

```diff
@@ -1121,7 +1121,10 @@ medical-imaging_Scenario_TaggingImageSets:
 medical-imaging_Scenario_ImageSetsAndFrames:
   title: Get started with &AHI; image sets and image frames using an &AWS; SDK
   title_abbrev: Get started with image sets and image frames
-  synopsis: import DICOM files and download image frames in &AHI;.<para>The implementation is structured as a workflow command-line application.</para>
+  synopsis: >
+   import DICOM files and download image frames in &AHI;.</para>
+    <para>The implementation is structured as a workflow command-line
+    application.</para><para>
```

Comment:
Couldn't you nix the ending close/open tag? Since the template has the close para. https://github.com/awsdocs/aws-doc-sdk-examples/blob/3ded2798e39c79c0b6adfd53f914afddae634ec9/.doc_gen/templates/zonbook/utility/synopsis.xml#L16

Reply:
not if I want a new line as requested by the guide writer,

Reply:
The elements would not balance. I have shown this to Casey and he approved it also.

Reply:
Yes, now I see you are right.